inc/options/cdn/azuremi.php

```diff
@@ -0,0 +1,122 @@
+<?php
+namespace W3TC;
+
+if ( ! defined( 'W3TC' ) ) {
+	die();
+}
+?>
+<tr>
+	<th style="width: 300px;"><label for="cdn_azuremi_user"><?php esc_html_e( 'Account name:', 'w3-total-cache' ); ?></label></th>
+	<td>
+		<input id="cdn_azuremi_user" class="w3tc-ignore-change" type="text"
+			<?php Util_Ui::sealing_disabled( 'cdn.' ); ?> name="cdn__azuremi__user" value="<?php echo esc_attr( $this->_config->get_string( 'cdn.azuremi.user' ) ); ?>" size="30" />
+	</td>
+</tr>
+<tr>
+	<th><label for="cdn_azuremi_clientid"><?php esc_html_e( 'Entra client id:', 'w3-total-cache' ); ?></label></th>
```

Comment:
`Entra Client ID`

Reply:
Changed it to `Entra client ID`. I see that sentence case is being used consistently all across.